### DISCUSSION TITLE
Hight load without useful work

### DISCUSSION THREAD

**Discussion by centralhardware** [photoprism/photoprism#2520] [Q&A]
I don't mind a high load, but not when I know in advance that no useful work will be done at all and when face recognition is run over and over again 24/7 there will be no useful work at all.
How i can completly disable face recognition when no image change?

**Comment by lastzero** [comment]
Could you provide more details? What did you do? What hardware, what version? Generally, no useless work is done, so you can also not disable doing useless work if that makes sense. You can disable face recognition completely though.

**Comment by centralhardware** [comment]
its log for two last days https://gist.github.com/centralhardware/68772b61dd5528e36476b39ff90cc482.  facial recognition start about 855 times and only 30 update anything. i think most of them were just running at the moment when I was working in Photoprism.  Is it really
necessary to run facial recognition every 15 minutes in the hope that something will change?
PS: i use latest docker version.

**Reply by lastzero** [reply]
At a minimum, it's updating the counts in the navigation.

For a lengthy discussion, see:

- https://github.com/photoprism/photoprism/issues/2182
